src/stuf/stuf_bg/tests/test_client.py

```diff
@@ -25,15 +25,15 @@
 class StufBGConfigTests(TestCase):
     def setUp(self):
         super().setUp()
-        self.service = StufServiceFactory.create()
+        self.service = StufServiceFactory.create(soap_service__soap_version="1.1")
```

Comment:
this one is sketchy - the default is SOAP 1.2, but the stuf BG service always emited `xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"` which is wrong according to the StUF-ZDS template conditional...

However, the assertion below about the content type expects the version to be 1.2. Either the `soapenv` or the content type assertion is wrong?

Reply:
Using 1.1 is okay, risk accepted. 1.1 is also what's used in practice.